pkg/handling/files.go

```diff
@@ -1 +1,56 @@
-package handling
\ No newline at end of file
+package handling
+
+import (
+	"io/ioutil"
+
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/dialog"
+	"fyne.io/fyne/v2/widget"
+)
+
+//opens a file dialog and loads the selected file's content into the editor.
+func OpenFile(window fyne.Window, editor *widget.Entry) {
+	dialog.ShowFileOpen(func(reader fyne.URIReadCloser, err error) {
+		if err != nil {
+			dialog.ShowError(err, window)
+			return
+		}
+		if reader == nil { //if no file is selected
+			return
+		}
+		defer reader.Close()
+
+		data, err := ioutil.ReadAll(reader)
```

Comment:
ioutil is deprecated in more recent versions of Go, so this should be changed to keep things clean
```suggestion
		data, err := io.ReadAll(reader)
```